components/ws-proxy/pkg/sshproxy/server.go

```diff
@@ -106,19 +108,45 @@ func init() {
 
 // New creates a new SSH proxy server
 
-func New(signers []ssh.Signer, workspaceInfoProvider proxy.WorkspaceInfoProvider, heartbeat Heartbeat) *Server {
+func New(signers []ssh.Signer, workspaceInfoProvider common.WorkspaceInfoProvider, heartbeat Heartbeat) *Server {
```

Comment:
Does it mean we now also will get metrics for SSH tunnel connections from ws-proxy?